packages/flowtip-react-dom/src/types.tsx

```diff
@@ -80,14 +71,12 @@ export type Props = {
   constrainBottom: boolean,
   /** Constrain the content at the bottom boundary. */
   constrainLeft: boolean,
-  children: React.Node,
+  children: React.ReactNode,
 
   render: Render,
 
-  children: React.Node,
-
   content?: React.ComponentType<ContentProps> | string,
-  tail?: ?React.ComponentType<TailProps>,
+  tail?: React.ComponentType<TailProps>,
```

Comment:
There is a semantic difference between `? :` and `?:`, the former is an optional property while the latter is a required property that is optionally `void`. I likely wrote it this way to accommodate strange flow behavior, but it's worth considering a little further.